docs/source/nbconvert_library.ipynb

```diff
@@ -511,27 +511,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
+    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
```

Comment:
Well things like that happen, It's ok. I think it show one of the limits of
GitHub that issues and comment are not decentralized.

On Sun, Jul 17, 2016 at 9:03 AM, Fernando Perez <notifications@github.com>
wrote:

> In docs/source/nbconvert_library.ipynb
> <https://github.com/jupyter/nbconvert/pull/338#discussion_r71082912>:
>
> > @@ -511,27 +511,18 @@
> >     "cell_type": "markdown",
> >     "metadata": {},
> >     "source": [
> > -    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
> > +    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
>
> Ouch! No, I am sure that was me know actually during the cleanup when I
> did create the attic. But I probably (obviously in error) thought that the
> new Jupyter nbconvert one had everything, and in this case did a full
> delete instead of moving it to the attic, like I did with many others. Oh
> well, mistakes happen... Not much we can do at this point, I guess.
> … <#m_-6890265982472960199_>
> On Sun, Jul 17, 2016, 01:33 Thomas Kluyver ***@***.***> wrote: In
> docs/source/nbconvert_library.ipynb <#338 (comment)
> <https://github.com/jupyter/nbconvert/pull/338#discussion_r71078849>>: >
> @@ -511,27 +511,18 @@ > "cell_type": "markdown", > "metadata": {}, >
> "source": [ > - "The following demonstration, as requested in [a GitHub
> issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235),
> adds the ability to exclude a cell by index. \n", > + "The following
> demonstration, as requested in [a GitHub issue](#137 (comment)
> <https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235>),
> adds the ability to exclude a cell by index. \n", Github audit logs say it
> was deleted by @fperez <https://github.com/fperez> <
> https://github.com/fperez>'s account last month:
> https://github.com/organizations/ipython/settings/audit-log?utf8=%E2%9C%93&q=action%3Arepo.destroy%20repo%3Aipython%2Fnbconvert
> Hopefully Fernando's just been so busy that he forgot this, and we don't
> have a really weird hacker. ;-) — You are receiving this because you were
> mentioned. Reply to this email directly, view it on GitHub <
> https://github.com/jupyter/nbconvert/pull/338/files/8c6af4062af857d13b8dfde3e412f907aacf41ef#r71078849>,
> or mute the thread <
> https://github.com/notifications/unsubscribe-auth/AADgMpoXf8MYOaL9TN-TvLSo4PG7kQh2ks5qWejPgaJpZM4JOEBa>
> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/jupyter/nbconvert/pull/338/files/8c6af4062af857d13b8dfde3e412f907aacf41ef#r71082912>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAUez5FHjFdCWqpXXy6zf3skrCbm8uHiks5qWjYrgaJpZM4JOEBa>
> .
>